atsign-foundation/at_server
# fix: Hive max length bug

fixes: https://github.com/atsign-foundation/at_server/issues/1863
**- What I did**
- Fixed issue in persistence layer for keys with length greater than max length suppported in hive(currently 255)
**- How I did it**
- Added max length check in put, putAll and create methods in HiveKeyStoreImpl
- added unit tests
**- How to verify it**
- units tests should pass
- point to persistence branch and run at_client and server tests
https://github.com/atsign-foundation/at_client_sdk/pull/1281
https://github.com/atsign-foundation/at_server/pull/1877
-------------------------
With this fix, server will return data:null when updating key with length > 255 chars. Have to make fix in server to return correct error message.
